microsoft/testsuites/cloud_hypervisor/ch_tests_tool.py

```diff
@@ -148,13 +219,71 @@ def _send_subtest_msg(
         environment: Environment,
         test_name: str,
         test_status: TestStatus,
+        test_message: str = "",
     ) -> None:
         subtest_msg = create_test_result_message(
-            SubTestMessage,
-            test_id,
-            environment,
-            test_name,
-            test_status,
+            SubTestMessage, test_id, environment, test_name, test_status, test_message
         )
 
         notifier.notify(subtest_msg)
+
+    def _list_perf_metrics_tests(self, hypervisor="kvm") -> List[str]:
+
+        self._log.debug("Listing the performance test cases")
+        tests_list = []
+        result = self.run(
+            f"tests --hypervisor {hypervisor} --metrics -- -- --list-tests",
+            timeout=self.TIME_OUT,
+            force_run=True,
+            cwd=self.repo_root,
+            no_info_log=False,
+            shell=True,
```

Comment:
```suggestion
            cwd=self.repo_root,
            shell=True,
```

Reply:
We are running other CH tests with no_info_log=False. Should we remove this for performance metrics test?